src/util/googleMaps.js

```diff
@@ -66,27 +66,32 @@ const predictionSuccessful = status => {
  * @param {String} search - place name or address to search
  * @param {String} sessionToken - token to tie different autocomplete character searches together
  * with getPlaceDetails call
+ * @param {Object} restriction - defines the component restrictions that can be used with
+ * the autocomplete service. Used to restrict search to specific country (or countries).
  *
  * @return {Promise<{ search, predictions[] }>} - Promise of an object
  * with the original search query and an array of
  * `google.maps.places.AutocompletePrediction` objects
  */
-export const getPlacePredictions = (search, sessionToken) =>
+export const getPlacePredictions = (search, sessionToken, restriction) =>
```

Comment:
It is probably better to rename this to `searchConfigurations` (or something like that) since there are more configurations that could be passed in than just `componentRestrictions`